server/pom.xml

```diff
@@ -86,6 +86,12 @@
             <artifactId>junit</artifactId>
             <scope>test</scope>
         </dependency>
+        <!--<dependency>
```

Comment:
Suggest remove commented section

Reply:
Whoops. You jts try something out and your shame is there for all to see.